test/test_library.py

```diff
@@ -416,11 +416,14 @@ def test_unicode_extension_in_fragment(self):
 
     def test_asciify_and_replace(self):
         config['asciify_paths'] = True
-        self.lib.replacements = [(re.compile(u'"'), u'q')]
+        self.lib.replacements = [
+            (re.compile(u'"'), u'q'),
+            (re.compile(os.path.sep), config['path_sep_replace'].get())
```

Comment:
Is this addition to the `replacements` necessary for this test? I don't quite see how it works…

It might also be nice to have a *separate* test just focused on the new functionality so we can see exactly what's being tested.